velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -142,5 +142,23 @@ TEST_F(DateTimeFunctionsTest, toUnixTimestamp) {
   EXPECT_THROW(evaluateOnce<int64_t>("to_unix_timestamp()"), VeloxUserError);
 }
 
+TEST_F(DateTimeFunctionsTest, makeDate) {
+  const auto makeDate = [&](std::optional<int32_t> year,
+                            std::optional<int32_t> month,
+                            std::optional<int32_t> day) {
+    return evaluateOnce<Date>("make_date(c0, c1, c2)", year, month, day);
+  };
+  Date d0;
+  parseTo("1920-01-25", d0);
+  EXPECT_EQ(makeDate(1920, 1, 25), d0);
+  EXPECT_FALSE(makeDate(std::nullopt, 7, 15).has_value());
```

Comment:
These tests are not needed as they test the framework, not the function. Let's remove.